index.bs

```diff
@@ -1658,6 +1680,9 @@ partial interface MLGraphBuilder {
     To <dfn>check clamp options</dfn> given {{MLClampOptions}} |options|, run the following steps:
   </summary>
     1. If |options|.{{MLClampOptions/minValue}} is greater than |options|.{{MLClampOptions/maxValue}}, then return false.
+
+        Issue(396): Not all implementations support {{MLClampOptions/minValue}} equal to {{MLClampOptions/maxValue}}.
```

Comment:
Hmm, is TF.js the only remaining outlier now? Every other ML library I know (include TF-main and XNNPack) support `min` and `max` fine. Well, there's always the decomposition `clamp(x) = min(max(x, maxValue), minValue)` too, but I'll open an issue at TF.js for better consistency across libraries (https://github.com/tensorflow/tfjs/blob/tfjs-v4.7.0/tfjs-core/src/ops/clip_by_value.ts#L47).